rapid_elastic/elastic_helper.py

```diff
@@ -1,11 +1,10 @@
 import dataclasses
 from typing import List
-from elasticsearch8 import Elasticsearch
+from elasticsearch import Elasticsearch
```

Comment:
We want elasticsearch8 here - elasticsearch recently released 9.0 which doesn't work with server 8.x - so we want to pin to 8.x until we upgrade the server side.

(I'll fix this, just saying)